examples/resources/bloxone_dhcp_ha_group/resource.tf

```diff
@@ -30,3 +36,20 @@ resource "bloxone_dhcp_ha_group" "example_tags" {
     location = "site1"
   }
 }
+
+resource "bloxone_dhcp_ha_group" "example_anycast" {
```

Comment:
The anycast example for HA group can be a separate file like `resource_with_anycast.tf` 

You will have to create a new template file in `templates/resources/bloxone_dhp_ha_group.md.tmpl` and add the `resource_with_anycast.tf` on to a different section.

Reply:
done